src/components/Text/Text.js

```diff
@@ -2,15 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import bem from 'bem';
 import styles from './Text.scss';
+import { SIZES } from '../../constants';
 
 const { block } = bem({
     name: 'Text',
     classnames: styles,
-    propsToMods: ['muted']
+    propsToMods: ['muted', 'size']
 });
 
 const Text = props => {
-    const { children, inline, ...rest } = props;
+    const { bold, children, inline, size, ...rest } = props;
```

Comment:
`bold`?

Reply:
I can add it if you think it should be configured through props but I vaguely recall you challenging that, hence I didn't add it yet